tests/cdiconfig_test.go

```diff
@@ -229,11 +227,6 @@ var _ = Describe("CDI ingress config tests, using manifests", func() {
 	)
 
 	BeforeEach(func() {
-		version := *semver.New(tests.GetKubeVersion(f))
```

Comment:
not needed anymore?

Reply:
It's testing for kubectl 1.14.0 (released in 2019), I assume nobody is running the testsuite with a kubectl that old.